.github/workflows/ci.yaml

```diff
@@ -271,6 +271,13 @@ jobs:
             echo "cover=false" >> $GITHUB_OUTPUT
           fi
 
+          # if macOS, install google-chrome for scaletests. As another concern,
+          # should we really have this kind of external dependency requirement
+          # on standard CI?
+          if [ "${{ matrix.os }}" == "macos-latest" ]; then
+            brew install google-chrome
+          fi
```

Comment:
Why are we installing a scaletest dependency in the regular `test-go` action?

Reply:
CI wasn't passing. Presumable google-chrome is already installed in the other operating systems.

Reply:
It looks like I unintentionally added this dependency via https://github.com/coder/coder/pull/9927. The unit test itself doesn't actually run chromedp but the setup part apparently does require it.